app/assets/stylesheets/components/_btn.scss

```diff
@@ -44,6 +45,20 @@
       width: 0;
     }
   }
+
+  &--l {
+    &::before {
+      border-bottom-left-radius: $rule-border-radius;
+      border-top-left-radius: $rule-border-radius;
```

Comment:
May save a couple of characters with shorthand, `border-radius: $rule-border-radius 0 0 $rule-border-radius` but not necessarily as readable as what you have.